docs/user-guide/running.md

```diff
@@ -68,7 +80,31 @@ Following step should be done after flotta-device-worker is installed on the edg
 
 Start yggdrasil with from the yggdrasil repository directory:
 
-`sudo go run ./cmd/yggd --log-level trace --transport http --cert-file /etc/pki/consumer/cert.pem --key-file /etc/pki/consumer/key.pem --client-id-source machine-id --http-server <your.k8s-ingress:8888>`
+`
+sudo yggd \
+  --protocol http \
+  --path-prefix api/flotta-management/v1 \
+  --client-id $(cat /etc/machine-id) \
+  --cert-file /tmp/cert.pem \
+  --key-file /tmp/key.pem \
+  --ca-root /tmp/ca.pem \
+  --socket-addr @yggd \
+  --server 127.0.0.1:8043
+`
+
+When running Yggdrasil, a new edgedevice should be added into the list
+
+```
+kubectl get edgedevices --all-namespaces
```

Comment:
Aren't we creating edgedevices in default namespace for now? Why not to use -n default?